hack/local-up-cluster.sh

```diff
@@ -184,6 +184,39 @@ cleanup()
   exit 0
 }
 
+function createUsers {
+
+### Example token file.  Obviously you wouldn't use this in production.
+cat << EOF > /tmp/kubehack-token
```

Comment:
better filename? `kube-apiserver-tokens`? can you generate a token to include here and in the kubeconfig file?